app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -1109,19 +1106,9 @@ private FormIndex retrieveAndValidateFormIndex(int sessionDescriptorId) {
         return null;
     }
 
-    private FormIndex deserializeFormIndex(String serializedFormIndex) {
-        if (serializedFormIndex != null) {
-            try{
-                byte[] decodedFormIndex = Base64.decode(serializedFormIndex, Base64.DEFAULT);
-                return SerializationUtil.deserialize(decodedFormIndex, FormIndex.class);
-            } catch(Exception e) {
-                Logger.exception("Deserialization of last form index failed ", e);
-            }
-        }
-        return null;
-    }
-
     private void handleFormLoadCompletion(AndroidFormController fc) {
+        HiddenPreferences.clearInterruptedFormState();
```

Comment:
is there any bheavior change caused by the change in position for this call ?

Reply:
Not necessarily, more for visibility. Initially though, I thought that this call could be prevented by [this return](https://github.com/dimagi/commcare-android/blob/f87f6e4863c6f868cfdf4adda7b25cc34285cfc7/app/src/org/commcare/activities/FormEntryActivity.java#L1146), which is triggered when saved forms are loaded, but because we restore the session immediately after the login, it seems unlikely.